engine/consensus/matching/core.go

```diff
@@ -382,7 +382,7 @@ func (c *Core) OnBlockFinalization() error {
 			lastSealed.ID(), lastSealed.Height, err)
 	}
 
-	c.log.Info().
+	c.log.Debug().
```

Comment:
I think this is a pretty interesting metric, since it shows when we don't have results for finalized blocks. Through, in the overwhelming amount of cases, we just log that there aren't any blocks we are requesting receipts for: https://github.com/onflow/flow-go/blob/a1a18ce4c01efbb19059221a7fea7b67f2c19a8d/engine/consensus/matching/core.go#L300

Maybe it is worth logging when `pendingReceiptRequests` is larger zero: 
```suggestion
	// this runs frequently, so we only log if there are pending requests
	if pendingReceiptRequests > 0 {
		c.log.Info().
		    ⋮
```

Reply:
https://github.com/onflow/flow-go/pull/7174/commits/1500acd3eb51cdb00f87f294b2defe9cb8e6177b